src/effects/presets/effectchainpresetmanager.cpp

```diff
@@ -593,10 +593,27 @@ void EffectChainPresetManager::resetToDefaults() {
     generateDefaultQuickEffectPresets();
     prependRemainingPresetsToLists();
 
+    // Re-add the empty chain preset
+    EffectChainPresetPointer pEmptyChainPreset = createEmptyChainPreset();
+    m_effectChainPresets.insert(pEmptyChainPreset->name(), pEmptyChainPreset);
+    m_quickEffectChainPresetsSorted.prepend(pEmptyChainPreset);
+
     emit effectChainPresetListUpdated();
     emit quickEffectChainPresetListUpdated();
 }
 
+EffectChainPresetPointer EffectChainPresetManager::createEmptyChainPreset() {
```

Comment:
doesn't need to be a member function. Can you make this a free function and move it to an anonymous namespace at the top of this file?

Reply:
sure.